tests/ices.zunit

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env zunit
+
+@setup {
+  ZPLUGINS=$ZINIT[PLUGINS_DIR]
+
+  function _zunit_assert_not_exists() {
```

Comment:
I think this assertion exists?

<img width="1367" alt="Screen Shot 2022-05-22 at 12 05 53" src="https://user-images.githubusercontent.com/10052309/169707048-6e7eeb32-6ff7-4005-9275-00480aaf59db.png">

Reply:
The assertion, as it is written, only checks if a file does exist and reports if it is not existing. I've neither found a way to reverse any assertion now would it make sense: the assertion would report a file missing (`echo "'$pathname' does not exist"` and then would get reversed.